### PR TITLE
added alternative database CouchDb

### PR DESCRIPTION
**Pull request by codezork**
Like for the MONGODB, you can provide a COUCHDB url which overrides the MongoDB usage. It is prepared in the .ev.example file and explained in README.md. Tests are untouched and still work against MongoDb.

### CODE REVIEW

File: src/server/api/cla.js [hunk 1]
```diff
@@ -582,7 +579,8 @@ async function prepareForValidation(item, user) {
     for (let i = needRemove.length - 1; i >= 0; --i) {
         user.requests.splice(needRemove[i], 1)
     }
-    user.save()
+    await userService.save(user);
+    //user.save()
```
Comment (ScottBrenner) [inline src/server/api/cla.js]:
```suggestion
```

File: src/server/app.js [hunk 2]
```diff
@@ -147,30 +152,58 @@ async.series([
...
                 collection: 'cookieSession'
             })
         }
         session.cookie.secure = app.get('env') === 'production'
-
         app.use(expressSession(session))
         app.use(passport.initialize())
         app.use(passport.session())
-
+
```
Comment (ScottBrenner) [inline src/server/app.js]:
```suggestion

```

**Comment by joerg-codecentric** [comment]
fixed deepscan issues - conflicts must be resolved
Jörg Riegel must be deleted from contributor list

**Comment by KharitonOff** [comment]
@joerg-codecentric great, thanks. If you add the email address, which was used for the commits, to your github account, then cla status will also switch to green.

**Comment by joerg-codecentric** [comment]
Had to force author change due to wrong commit history to get rid of Jörg Riegel

**Comment by KharitonOff** [comment]
Closing this PR due to unresolved conflicts...
For further development on this feature, please create a PR for the cla-assistant/Daimler-master branch. We will keep this branch in sync with master.

**Pull request closed without merging**